### PR TITLE
✨(storage): added data-access library, Firebase and Storage Repositories

### PR DESCRIPTION
**Pull request by LarisaBothaTuks**
Created the repository directories and added classes to directly communicate with external sources ie Firebase and the database.

### CODE REVIEW

**Review by lindzik — COMMENTED** [review]
There seem to be no conflicting files and no errors.

**Review by lindzik — APPROVED** [review]

**Review by arneschreuder — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: libs/api/storage/repository/data-access/src/lib/FirebaseRepository.repository.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+different folder. Please see below. The firebaseConfig is currently connected to one 
+personal firebase projects will be updated accordingly later on. If you require external
+storage please contact me (Larisa-082 796 0342) I will simply add a folder to the firebase
+bucket and add a upload function then you can use this class to manage and organise your 
+own files*/
+
+
+@Injectable()
+export class FirebaseService {
+  firebaseConfig = {
```
Comment (arneschreuder) [inline libs/api/storage/repository/data-access/src/lib/FirebaseRepository.repository.ts:16]: Please move this to environments in apps/api or use a nestjs config service.  This is fine for now, we can merge in and fix later. But these keys should not be in the repo, people can then copy them for their own use. Remember, this project is public. So these config stuff needs to be coordinated with devops using env secrets

**Review by arneschreuder — CHANGES_REQUESTED** [review]

**Comment by arneschreuder** [comment]
> There seem to be no conflicting files and no errors.

Are we looking at the same PR? 😂

File: libs/api/storage/repository/data-access/src/lib/FirebaseRepository.repository.ts (hunk 1 above)
Comment (LarisaBothaTuks) [reply libs/api/storage/repository/data-access/src/lib/FirebaseRepository.repository.ts:16]: Noted Thank you.

**Comment by lindzik** [comment]
Oh i apologise i did not notice the conflicts. From my knowledge the conflicts are there because the files(package.json and yarn.lock) from the initial PR are older versions than the ones on the develop branch. Would that be a correct assumption and an accurate place to work on the conflicts

**Pull request closed without merging**